### DISCUSSION TITLE
Content scripts with yarn pnp?

### DISCUSSION THREAD

**Discussion by clhuang** [tailwindlabs/tailwindcss#11096] [Help]
We have a monorepo using yarn pnp, which means that there is no node_modules folder and that modules are loaded from a centralized set of zip files on the fly.

When using a package such as https://github.com/onesine/react-tailwindcss-datepicker#readme, which requires specifying something like this in the config file: 
```
// in your tailwind.config.js
module.exports = {
    // ...
    content: ["./src/**/*.{js,jsx,ts,tsx}", "./node_modules/react-tailwindcss-datepicker/dist/index.esm.js"],
    // ...
}
```

what do people instead do when using PnP?

**Comment by ybelakov** [comment]
Same issue here with a NextJS project
(https://github.com/tailwindlabs/tailwindcss/discussions/11118)

**Comment by kachar** [comment]
I'm having the same problem with yarn pnp

Still haven't found a way to include plugins and libs like `preline`

**Comment by lxia1220** [comment]
@clhuang @ybelakov @kachar  You can find a workaround here.
https://github.com/tailwindlabs/tailwindcss/discussions/11118#discussioncomment-6082657
